x-pack/plugins/monitoring/public/components/page_loading/index.js

```diff
@@ -45,3 +46,18 @@ export function PageLoading() {
     </EuiPage>
   );
 }
+
+function PageLoadingTracking({ pageViewTitle }) {
+  const path = pageViewTitle.toLowerCase().replace(/-/g, '').replace(/\s+/g, '_');
+  console.log({ pageViewTitle, path });
+  useTrackPageview({ app: 'stack_monitoring', path });
```

Comment:
Can these paths include dynamic names? You may not want to track every single unique path you can, but rather just a subset of normalized ones without dynamic path segments (esp if it has query strings in it). 

I also recommend throwing in a 2nd track pageview but with a `delay: 15000` on the call, which shows how many clusters stayed on the page some "significant amount of time" and will allow your page views to be compared to other observability ones which are currently all doing that. (We can eventually normalize that so it happens with one call across the board consistently, but for now we are keeping it all very manual.)

Reply:
@jasonrhodes Great idea. I was wondering why that was there for the examples I went through.

In terms of dynamic data, this page titles will not contain that. I went through all the unique instance/node pages and ensured the tracking name just said something like `kibana_instance`. With that said, we still have the beats/apm pages in legacy formats, which I hope to resolve in https://github.com/elastic/kibana/pull/76946 before updating here